src/pages/use-cases/network-policy.jsx

```diff
@@ -138,7 +139,22 @@ const testimonials = [
       },
     ],
     description:
-      'G DATA chose Cilium as their preferred solution for networking, observability, and security to benefit from its enhanced network policy visibility. Cilium allows their development teams to self service network policies without needing to involve the IT team.',
+      '"In the beginning, it was hard for our developers to write network policies because we were in our early Kubernetes adoption phase. Everyone had to learn a lot of stuff in Kubernetes and then also had to learn how to write network policies. Cilium helped reduce the mental overhead and helped speed up our development process so that we can bring new features to customers faster."',
+    quote: true,
+    person: 'Jan Jansen, Platform Engineer, G DATA',
+  },
+  {
+    logo: WSO2Logo,
+    title: 'Implementing Zero Trust Security with Cilium',
+    CTAs: [
+      {
+        CTAtext: 'Read Case Study',
+        url: 'https://www.cncf.io/case-studies/ws02/',
+      },
+    ],
+    description: `"We migrated all our clusters to Cilium, enhancing our cell-based architecture with Cilium's advanced Layer 3 and Layer 4 network policies. Our setup also includes Hubble, integrated with Layer 3 policies for observability within the cell. We encrypted our network traffic with Cilium's IPsec transparent encryption, ensuring all our outbound data transfers are encrypted. Cilium's transparent encryption distinctive advantage lies in its kernel-level operation which provides it with a significant edge in performance over user-space solutions."`,
```

Comment:
this quote has too many other things besides network policy. Can you cut is down or just summarize what they did?

Reply:
okay